cypress/support/client.ts

```diff
@@ -57,6 +62,19 @@ declare global {
     }
 }
 
+Cypress.Commands.add("setUpKeyBackup", (): Chainable<void> => {
+    cy.get('.mx_AccessibleButton[aria-label="User menu"]').click();
+    cy.get('.mx_AccessibleButton[aria-label="Security & Privacy"]').click();
+    cy.contains(".mx_AccessibleButton", "Set up Secure Backup").click();
+    cy.contains(".mx_Dialog_primary", "Continue").click();
+    cy.get(".mx_CreateSecretStorageDialog_recoveryKey code").invoke("text").as("securityKey");
+    cy.contains(".mx_AccessibleButton", "Copy").click();
+    cy.contains(".mx_Dialog_primary:not([disabled])", "Continue").click();
+    cy.contains(".mx_Dialog_title", "Setting up keys").should("exist");
+    cy.contains(".mx_Dialog_title", "Setting up keys").should("not.exist");
```

Comment:
Can we do this within a `within` block/s to better approximate limitations we impose on real users (which can't click on elements outside of a modal without closing it unlike this synthetic user)